src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -315,67 +304,42 @@ Background="{DynamicResource {x:Static nuget:Brushes.DetailPaneBackground}}"
-                x:Name="_targetFramework"
-                AutomationProperties.Name="{Binding Text}"
-                Visibility="{Binding TargetFramework,Converter={StaticResource NuGetFrameworkToVisibilityConverter}}" />
-              <ItemsControl
-                ItemsSource="{Binding Dependencies}"
-                IsTabStop="False"
-                AutomationProperties.LabeledBy="{Binding ElementName=_targetFramework}">
-                <ItemsControl.ItemTemplate>
-                  <DataTemplate>
-                    <TextBox
-                      x:Name="_dependencies"
-                      Style="{DynamicResource SelectableTextBlockStyle}"
-                      AutomationProperties.Name="{Binding Text}"
-                      Text="{Binding Mode=OneWay}"/>
-                  </DataTemplate>
-                </ItemsControl.ItemTemplate>
-              </ItemsControl>
-              <TextBlock
-                FontStyle="Italic"
-                Text="{x:Static nuget:Resources.Text_NoDependencies}"
-                x:Name="_noDependencies"
-                AutomationProperties.Name="{Binding Text}"
-                Visibility="{Binding Dependencies,Converter={StaticResource EmptyEnumerableToVisibilityConverter}}" />
-            </StackPanel>
-          </DataTemplate>
-        </ItemsControl.ItemTemplate>
-      </ItemsControl>
+    <!-- dependencies treeview -->
+    <TreeView
+        x:Name="_dependencies"
+        Grid.Row="3"
+        Margin="-21,0,0,0"
+        BorderThickness="0"
+        Background="{x:Null}"
+        TreeViewItem.Selected="OnItemSelected"
+        LostFocus="TreeView_LostFocus"
+        AutomationProperties.Name="{x:Static nuget:Resources.Label_Dependencies}">
+      <TreeView.Template>
+        <!--
+          Replaces default control template for this treeview with a slimmer control template.
+          Specifically, this one has no ScrollViewer...which wasn't being used, and was making mousewheel not work for
+          the larget detail scrollable area while the mouse was over the treeview.
+        -->
+        <ControlTemplate TargetType="TreeView">
+          <ItemsPresenter />
+        </ControlTemplate>
+      </TreeView.Template>
+      <TreeView.Resources>
+        <Style TargetType="TreeViewItem">
+          <Setter Property="IsExpanded" Value="true" />
+          <Setter Property="AutomationProperties.Name" Value="" />
+          <Setter Property="Foreground" Value="{DynamicResource {x:Static nuget:Brushes.UIText}}" />
+          <Setter Property="FontWeight" Value="normal" />
+        </Style>
+        <HierarchicalDataTemplate ItemsSource="{Binding Dependencies}" DataType="{x:Type nuget:PackageDependencySetMetadata}">
+          <TextBlock Text="{Binding TargetFrameworkDisplay}" Padding="0,2,0,0" />
+        </HierarchicalDataTemplate>
+        <DataTemplate DataType="{x:Type nuget:PackageDependencyMetadata}">
+          <TextBlock Text="{Binding }" ToolTip="{Binding }" />
```

Comment:
Is the extra space significant?  `{Binding }` vs. `{Binding}`?

Reply:
nope identical. markup extensions (like binding), can have params. but this doesn't. likely cleaner to remove space.